docs/content/operations/dashboard.md

```diff
@@ -50,6 +50,25 @@ To enable the dashboard, you need to enable Traefik's API.
       dashboard = true
     ```
     
+    ```yaml tab="File (YAML)"
+    api:
+      # Dashboard
+      #
+      # Optional
+      # Default: true
+      #
+      dashboard: true
+    ```
+    
+    ```bash tab="CLI"
```

Comment:
WDYT about removing the previous admonition named "Using the Command Line", 
as it duplicates the information here?